ProGetAutomation/Functions/Get-ProGetAssetContent.ps1

```diff
@@ -0,0 +1,42 @@
+
+function Get-ProGetAssetContent
+{
+    <#
+    .SYNOPSIS
+    Gets the content of an asset in an asset directory.
+
+    .DESCRIPTION
+    The `Get-ProGetAssetContent` function gets an asset's content from ProGet. Pass the name of the root asset directory to the `DirectoryName` parameter. Pass the path to the asset to the `Path` parameter. If the URL to an asset directory in ProGet is `https://proget.example.com/assets/versions/subdirectory/file`, the directory parameter is the first directory after `assets/` (in this example `versions`). The `Path` parameter would be the rest of the url in this case `subdirectory/file`. 
+
+    If an asset doesn't exist, an error will be written and nothing is returned.
+
+    Pass a ProGet session object to the `$Session` parameter. This object controls what instance of ProGet to use and what credentials and/or API keys to use. Use the `New-ProGetSession` function to create session objects.

```

Comment:
Nit: To be consistent, I think we usually don't include a `$` with parameter names in docs. e.g. See parameters specified in line 9.

Reply:
@jlariono Fixed. Ready for re-review.